common/lib/xmodule/xmodule/course_module.py

```diff
@@ -390,44 +390,46 @@ class CourseFields(object):
         scope=Scope.settings,
         default=[
             {
-                "short_description": _("Getting Started With Studio"),
+                "short_description": _("Course Introduction"),
                 "items": [
                     {
-                        "short_description": _("Add Course Team Members"),
+                        "short_description": _("Welcome Message"),
                         "long_description": _(
-                            "Grant your collaborators permission to edit your course so you can work together."
+                            "Provide a welcome message on course info page at the beginning of the course."
```

Comment:
Welcome learners with a message on the Course Info page, the first page shown when your course is accessed.